homeassistant/components/lamarzocco/strings.json

```diff
@@ -36,6 +36,23 @@
     }
   },
   "entity": {
+    "sensor": {
+      "current_temp_coffee": {
+        "name": "Current coffee temperature"
+      },
+      "current_temp_steam": {
+        "name": "Current steam temperature"
+      },
+      "drink_stats_coffee": {
+        "name": "Drink statistics coffee"
```

Comment:
Please be a bit more descriptive, this sounds very cryptic.

Total coffees made? Something like that